inttest/terraform/test-cluster/main.tf

```diff
@@ -14,6 +14,7 @@ resource "aws_key_pair" "cluster-key" {
 
 // Save the private key to filesystem
 resource "local_file" "aws_private_pem" {
+  file_permission = "700"
```

Comment:
If you want to use this key from the terminal this permission level is required. Otherwise it will not connect to the server.